integration-test/common/run_sweep_tests.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+import os
+import subprocess
+
+def test_point(run_script):
+  process = subprocess.Popen(["bash", os.path.basename(run_script)],
+                             cwd=os.path.dirname(run_script),
+                             stdout=True,
+                             stderr=subprocess.PIPE)
+  _, stderr = process.communicate()
```

Comment:
Why do we not care about stdout?

Reply:
Because in `run.sh`, it redirects outputs to two files: stdout and stderr, so we won't see anything printed to stdout.

Reply:
But we might see stdout in the future. If there's nothing here, all you'll get is just an extra empty line, so there's no harm in adding it.

Reply:
Sure. Added the stdout.